docs/reference/autoscaling/deciders/machine-learning-decider.asciidoc

```diff
@@ -3,47 +3,46 @@
 [[autoscaling-machine-learning-decider]]
 === Machine learning decider
 
-The {ml} decider (`ml`) calculates the memory required to run
-{ml} jobs created by users.
+The {ml} decider (`ml`) calculates the memory required to run {ml} jobs.
 
 The {ml} decider is enabled for policies governing `ml` nodes.
 
 NOTE: For {ml} jobs to open when the cluster is not appropriately
-scaled, `xpack.ml.max_lazy_ml_nodes` should be set to the largest
-number of possible {ml} jobs (see <<advanced-ml-settings>>). In
-{ess} this is already handled.
+scaled, set `xpack.ml.max_lazy_ml_nodes` to the largest number of possible {ml} 
+jobs (refer to <<advanced-ml-settings>> for more info). In {ess} this is 
+automatically set.
 
 [[autoscaling-machine-learning-decider-settings]]
 ==== Configuration settings
 
-Both `num_anomaly_jobs_in_queue` and `num_analytics_jobs_in_queue`
-are designed to be used to delay a scale-up event. They indicate how many jobs
-of that type can be unassigned from a node due to the cluster being
-too small. Both settings are only considered for jobs that could
-eventually be fully opened given the current scale. If a job is too
-large for any node size or if a job couldn't ever be assigned without
-user intervention (for example, a user calling `_stop` against a real-time 
+Both `num_anomaly_jobs_in_queue` and `num_analytics_jobs_in_queue` are designed 
+to delay a scale-up event. They indicate how many jobs of that type can be 
+unassigned from a node due to the cluster being too small. Both settings are 
+only considered for jobs that can be opened given the current scale. If a job is 
+too large for any node size or if a job can't be assigned without user 
+intervention (for example, a user calling `_stop` against a real-time 
 {anomaly-job}), the numbers are ignored for that particular job.
 
 `num_anomaly_jobs_in_queue`::
 (Optional, integer)
-Number of queued anomaly jobs to allow. Defaults to `0`.
+Specifies the number of queued {anomaly-jobs} to allow. Defaults to `0`.
 
 `num_analytics_jobs_in_queue`::
 (Optional, integer)
-Number of queued analytics jobs to allow. Defaults to `0`.
+Specifies the number of queued {dfanalytics-jobs} to allow. Defaults to `0`.
 
 `down_scale_delay`::
 (Optional, <<time-units,time value>>)
-Delay before scaling down. Defaults to 1 hour. If a scale down is possible
-for the entire time window, then a scale down is requested. If the cluster
-requires a scale up during the window, the window is reset.
+Specifies the time to delay before scaling down. Defaults to 1 hour. If a scale 
+down is possible for the entire time window, then a scale down is requested. If 
```

Comment:
> down is possible for the entire time window...

This is probably a question for @benwtrent, but something about this description is unclear IMO. Is it saying that it somehow knows in advance that a scale up is required within the specified time interval and if so, won't scale down?

Reply:
> Is it saying that it somehow knows in advance that a scale up is required within the specified time interval and if so, won't scale down?

No, the deciders are periodically polled for scaling decisions. If during all these polls (over the span of 1 hour), a scale down is possible, then a scale down will be requested.

Reply:
If I understand correctly, then it says that if a scale-up is not required during the time of `down_scale_delay`, then it will scale down after the time is up. If we use the default value for example, then if scale-up is not needed during one hour, then it scales down after one hour. If scale-up is needed, then the timer starts again.

Reply:
yes